roles/user-interface/tasks/main.yml

```diff
@@ -1 +1,58 @@
 ---
+- name: Install login_checks.sh script.
+  copy:
+    src: login_checks.sh
+    dest: /etc/pam-script.d/login_checks.sh
+    owner: root
+    group: root
+    mode: '0755'
+
+- name: Deploy password-auth-ac for PAM.
```

Comment:
handler is not required.

Reply:
I struggle to follow. Perhaps you can explain this when you're here.